frontend/packages/local-storage-operator-plugin/src/utils/index.ts

```diff
@@ -15,23 +15,19 @@ export const getNodes = (
   selectedNodes: string[],
 ): string[] => (showNodes ? selectedNodes : allNodes);
 
-export const getLabelIndex = (
-  nodeSelector: NodeAffinityTerm[],
-  label: string,
-  operator: string,
+export const getNodeSelectorTermsIndices = (
+  nodeSelectorTerms: { matchExpressions: MatchExpression[]; matchFields: MatchExpression[] }[],
 ) => {
   let [selectorIndex, expIndex] = [-1, -1];
 
-  _.forEach(nodeSelector, (selector, index) => {
+  _.forEach(nodeSelectorTerms, (selector, index) => {
```

Comment:
```suggestion
  nodeSelectorTerms?.forEach((selector, index) => {
```

Reply:
If I have the defaults set then I can skip optional chaining.